### DISCUSSION TITLE
Loopback Connector vs TypeORM Module

### DISCUSSION THREAD

**Discussion by tokidoki11** [loopbackio/loopback-next#9346] [Q&A] [answered]
I would like to ask the question regarding the future of the Loopback connector vs typeORM module.

Background
---
Currently im contemplating whether to kickstart a new project with Loopback Connector MySQL or typeORM Module

Current Loopback MySQL Connector is not supporting current LTS node version 18, and version 16 will be ending on september 2023 (approximately half year to go)
https://github.com/loopbackio/loopback-connector-mysql/issues/477

Since TypeORM is being worked in the loopback-next repository i wonder whether for the future typeORM will be the suggested implemetation rather than connector (or even move away from connector completely)

To think of it, TypeORM will separate the DB driver to the loopback project itself thus granting more customization to the DB connection

Has anyone got any idea regarding the future of both modules? and any suggestions is greatly appreciated!

**Comment by shubhamp-sf** [comment]
Although the maintainers of the framework are better equipped to provide clarity on the future potential of the modules you mentioned, I would like to supplement your knowledge with an additional option: [@loopback/sequelize](https://www.npmjs.com/package/@loopback/sequelize). It is a most recent addition to the official loopback extensions, much like the typeorm module you mentioned, and was developed by me. Therefore, I can assure you that it will be actively maintained in the foreseeable future.

**Comment by tokidoki11** [comment, marked as answer]
FYI
I went with loopback typeorm but hand in hand with the main typeorm too for migration and etc

it seems mysql8 has been supported too
